back/app/policies/project_review_policy.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class ProjectReviewPolicy < ApplicationPolicy
+  def create?
+    return false unless active?
+
+    can_moderate_project? && record.requester == user
```

Comment:
Wondering a bit whether can moderate project includes admins and folder moderators because they, by definition can also moderate the project but I am not sure they should be able to create a review request?

Reply:
Yes, they are included on purpose. The goal in the future would be to allow an admin to ask a review to another (specific) admin.